app/main/index.js

```diff
@@ -103,7 +103,7 @@ function checkConnectivity() {
 function checkConnection() {
 	// eslint-disable-next-line no-unused-vars
 	mainWindow.webContents.on('did-fail-load', (event, errorCode, errorDescription, validatedURL) => {
-		if (errorDescription === 'ERR_INTERNET_DISCONNECTED' || errorDescription === 'ERR_PROXY_CONNECTION_FAILED') {
+		if (errorDescription === 'ERR_INTERNET_DISCONNECTED' || errorDescription === 'ERR_PROXY_CONNECTION_FAILED' || errorDescription === 'ERR_CONNECTION_RESET' || errorDescription === 'ERR_NAME_NOT_RESOLVED') {
```

Comment:
Code clean up? Save all possible errors into an array and then match with errorDescription? Something like - 
```js
		const connectivityERR = [
			'ERR_INTERNET_DISCONNECTED',
			'ERR_PROXY_CONNECTION_FAILED',
			'ERR_CONNECTION_RESET',
			'ERR_NOT_CONNECTED'
		];

		if (errors.indexOf(errorDescription) >= 0) {
			console.error('error', errorDescription)
			checkConnectivity();
		}